docs/apache-airflow/macros-ref.rst

```diff
@@ -35,28 +35,23 @@ in all templates
 =====================================   ====================================
 Variable                                Description
 =====================================   ====================================
-``{{ ds }}``                            the execution date as ``YYYY-MM-DD``
-``{{ ds_nodash }}``                     the execution date as ``YYYYMMDD``
-``{{ prev_ds }}``                       the previous execution date as ``YYYY-MM-DD``
-                                        if ``{{ ds }}`` is ``2018-01-08`` and ``schedule_interval`` is ``@weekly``,
-                                        ``{{ prev_ds }}`` will be ``2018-01-01``
-``{{ prev_ds_nodash }}``                the previous execution date as ``YYYYMMDD`` if exists, else ``None``
-``{{ next_ds }}``                       the next execution date as ``YYYY-MM-DD``
-                                        if ``{{ ds }}`` is ``2018-01-01`` and ``schedule_interval`` is ``@weekly``,
-                                        ``{{ next_ds }}`` will be ``2018-01-08``
-``{{ next_ds_nodash }}``                the next execution date as ``YYYYMMDD`` if exists, else ``None``
-``{{ yesterday_ds }}``                  the day before the execution date as ``YYYY-MM-DD``
-``{{ yesterday_ds_nodash }}``           the day before the execution date as ``YYYYMMDD``
-``{{ tomorrow_ds }}``                   the day after the execution date as ``YYYY-MM-DD``
-``{{ tomorrow_ds_nodash }}``            the day after the execution date as ``YYYYMMDD``
-``{{ ts }}``                            same as ``execution_date.isoformat()``. Example: ``2018-01-01T00:00:00+00:00``
+``{{ logical_date }}``                  the logical date of the DAG run (`pendulum.Pendulum`_)
+``{{ ds }}``                            the logical date as ``YYYY-MM-DD``
```

Comment:
I think this should be derrived from data_interval_start, rather than logical date.

Previously the implicit data interval was exection_date...next_exectuion_date

Reply:
How about `@once` DAGs? We decided to set its data interval to None, should this also become None?

Reply:
Hmmmmmm, _Probably_, but would make it breaking change :(

Reply:
Maybe we need to leave this as it is but deprecate it, and replace with `{{ data_interval_start | ds }}` (i.e. add new `ds`, `ds_nodash`, `ts`, etc filters) -- separate PR there.